package.json

```diff
@@ -57,14 +57,15 @@
     "zx": "^7.2.1"
   },
   "scripts": {
-    "dev": "turbo dev --filter=@clerk/* --filter=!@clerk/expo",
-    "build": "turbo build --concurrency=${TURBO_CONCURRENCY:-2}",
-    "test": "turbo test --concurrency=${TURBO_CONCURRENCY:-2}",
-    "test:ci": "turbo test --concurrency=${TURBO_CONCURRENCY:-2}",
+    "dev": "FORCE_COLOR=1 turbo dev --filter=@clerk/* --filter=!@clerk/expo",
```

Comment:
With `FORCE_COLOR=1` Turborepo outputs the logs with their colors. Makes it easier to read them